target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -802,14 +802,20 @@ tl-wr1043nd-v2)
 	ucidef_set_led_usbdev "usb" "USB" "tp-link:green:usb" "1-1"
 	ucidef_set_led_wlan "wlan" "WLAN" "tp-link:green:wlan" "phy0tpt"
 	;;
-tl-wr1043nd-v4)
-	ucidef_set_led_usbdev "usb" "USB" "tp-link:green:usb" "1-1"
+tl-wr1043nd-v4|\
+tl-wr1043n-v5)
```

Comment:
This doesn't look for me like a correct alphabetical order.
Please, fix it here and in all other places.

Reply:
Thanks for the review!
I prefer the natural sorting order, because its the one that comes after the 1043v4 - damn tpl for changing their antennas!
anyway - I hope it's fixed in all files :smile: